grails-shell/build.gradle

```diff
@@ -20,7 +27,7 @@ dependencies {
     api "org.fusesource.jansi:jansi:$jansiVersion"
     api "jline:jline:$jlineVersion"
     api "org.gradle:gradle-tooling-api:$gradleToolingApiVersion"
-    compileOnly "org.apache.groovy:groovy-templates:$groovyVersion"
+
     api "org.apache.groovy:groovy-xml:$groovyVersion"
```

Comment:
How about this one?

Reply:
Yes, I left this one as org.apache.groovy and 4.0.x since it was not for compilation and is needed by at least one project down the line that is running it with Groovy 4 (not grails-gradle-plugin)

We did the same on https://github.com/bertramdev/asset-pipeline/blob/5.0.x/asset-pipeline-core/build.gradle#L56-L74

Reply:
Ok, but the one in `asset-pipeline` is `testImplementation`. This one is `api`. Doesn't `api` put it on the `compileClasspath`?